### PR TITLE
Troll deterrent

### PR DESCRIPTION
**Pull request by JAremko**
Just a Friendly reminder to the green skin vandals. :thinking:

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -9,6 +9,15 @@ assignees: ''
 
 <!--
 
+!!!!!IF YOU ARE A TROLL READ THIHS FIST!!!!
```
Comment (Drarok) [inline .github/ISSUE_TEMPLATE/feature_request.md]: THIHS? FIST? 🤔

Comment (Snevzor) [reply .github/ISSUE_TEMPLATE/feature_request.md]: I hate these kind of typoes!

Comment (JAremko) [reply .github/ISSUE_TEMPLATE/feature_request.md]: me to :sob:

**Comment by oliversalzburg** [comment]
I am not a troll and this is the first thing I read in the source. How can I revert?

**Comment by JAremko** [comment]
:do_not_litter:  :put_litter_in_its_place:  :recycle:

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (NachoNav) [reply .github/ISSUE_TEMPLATE/feature_request.md]: I'm gonna need to make a pull request fixing that typo.

**Pull request closed without merging**